packages/ndla-ui/src/RelatedArticleList/RelatedArticleList.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import BEMHelper from 'react-bem-helper';
+import SafeLink from '../common/SafeLink';
+import { Document } from '../icons';
+
+const classes = new BEMHelper({
+  name: 'related-articles',
+  prefix: 'c-',
+});
+
+export const RelatedArticle = ({ title, introduction, to }) => (
```

Comment:
Hvor spesifiseres det hvilket ikon som skal vises?

Reply:
Var ikke helt ferdig 😉 . Skal være støtte for det nå, men det betyr at det er article-converter som må ha en mapping mellom det vi får fra taksonomitjenesten til ikon og klassenavn.  Har noe lignende i ndla-frontend https://github.com/NDLANO/ndla-frontend/blob/master/src/components/getResourceTypeMetaData.jsx